apps/webapp/src/components/header/navbar.tsx

```diff
@@ -7,25 +7,25 @@ export const Navbar = () => {
   const pathname = usePagePath();
 
   return (
-    <nav className='hidden gap-4 xl:flex'>
+    <nav className='hidden gap-4 xl:flex xl:grow xl:justify-between max-w-2xl'>
```

Comment:
The changes in this file make the nav bar a bit more flexible for smaller screen sizes.